avatars/avatars.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="author" content="Parth Pandya" />
+    <meta name="description" content="Avatars Docs" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <title>Avatar</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <h1 class="head-start">FoundOve UI</h1>
+      </nav>
+
+      <div class="pg-section">
+        <h2>Avatar</h2>
+        <hr />
+
+        <ul>
+          <li>Rounded Avatar of different sizes</li>
+          <div class="disp-avatar">
+            <img
+              class="avatar avatar-xl round"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+            <img
+              class="avatar avatar-md round"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+            <img
+              class="avatar avatar-sm round"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+          </div>
+
+          <li>Square Avatar of different sizes</li>
+          <div class="disp-avatar">
+            <img
+              class="avatar avatar-xl square"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+            <img
+              class="avatar avatar-md square"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+            <img
+              class="avatar avatar-sm square"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+          </div>
+
+          <li>Textual Avatars</li>
+          <div class="disp-avatar">
+            <p class="avatar avatar-xl round txt-avatar">P</p>
```

Comment:
The properties of avatar are overwritten by avatar-xl. They both have same props. Use only one of them. Do the same for other elements as well.

Also, in mobile screen, the text P is oval and not round. Check that too.

Reply:
Sure, I'll update it now.